### DISCUSSION TITLE
Sleep not working

### DISCUSSION THREAD

**Discussion by Kuimera** [rocksdanister/lively#1107] [General]
Hi, 

I'm having issues with sleep/standby, with Lively on my pc wont go to sleep.. I check your solution, and the video is set to mpv as instructed, still, it won't go to sleep. 

Any ideas?

Thank you.

**Comment by Kuimera** [comment]
Ok I figure it out, using the command line **powercfg /requests**

![issue](https://user-images.githubusercontent.com/7859537/157274907-28a97ca4-e6aa-4b21-8b05-72e4f900fe81.png)

the video/wallpaper I was using had sound in it and the sound driver was not letting the computer to go to sleep, I removed the sound from the video and it's all fine now

![clear](https://user-images.githubusercontent.com/7859537/157275356-17a4af3d-3048-4e2c-9d1f-86ea75ac03ce.png)

**Reply by rocksdanister** [reply]
ah that makes sense.. I am not sure if this is something that can be addressed from my end without having to mute mpv player.

**Reply by Kuimera** [reply]
> 

Thank you for the reply @rocksdanister , maybe an option in setting to disable soundcard for the program? Cause I tried the option to mute on the edit video and does not seem to work.

**Reply by rocksdanister** [reply]
In mpv there is option to disable track which in theory should work.. the thing is many need wallpaper with audio ugh..

Also moved discussion to issue https://github.com/rocksdanister/lively/issues/1109
